lib/test/test_hamburger.rb

```diff
@@ -150,7 +138,7 @@ def test_hamburger_content_student
 
   def test_hamburger_content_nobody
     contents = Hamburger.get_hamburger_contents({level: nil, script_level: nil, user_type: nil, language: "en"})
-    assert_includes_id contents[:entries], "hamburger-districts"
```

Comment:
This link was removed with the `signed_out_entries` list so I replaced it with the `learn` link since that's the first to show up in the list.